doc/rbd/iscsi-initiators.rst

```diff
@@ -12,6 +12,7 @@ Configuring the iSCSI Initiators
 
     .. warning::
 
+        The Ceph iSCSI gateway does not support reservations in any scenario.
```

Comment:
Please drop this -- I don't think `doc/rbd/iscsi-initiators.rst` needs any changes.
```suggestion
```

Reply:
I've dropped it.